streams/ec.m3u

```diff
@@ -35,24 +35,44 @@ https://stmv1.voxplayer.com.br/vidieg12/vidieg12/playlist.m3u8
 https://go.majestadtv.ec/hls/stream.m3u8
 #EXTINF:-1 tvg-id="Manavision.ec",Manavisión (480p)
 https://play.once.net.ec/play/Ily2oYdd3E1fi9wK2zORoRovFU9P5xM0g3CNVK2MT0M/.m3u8
+#EXTINF:-1 tvg-id="MarcaTV.ec",Marca TV (1080p)
+https://eu1.servers10.com:8081/8030/index.m3u8
+#EXTINF:-1 tvg-id="MarioPintoTV.ec",Mario Pinto TV (720p) [Not 24/7]
+https://eu1.servers10.com:8081/8028/index.m3u8
+#EXTINF:-1 tvg-id="MetropoliMediosTV.ec",Metropoli Medios TV (720p)
+https://eu1.servers10.com:8081/8214/index.m3u8
 #EXTINF:-1 tvg-id="MulticanalCatamayo.ec",Multicanal Catamayo (720p) [Not 24/7]
 https://multicanal.streamseguro.com/hls/streaming.m3u8
 #EXTINF:-1 tvg-id="Nortvision.ec",Nortvisión (720p)
 https://plataformavideo.com/live/nortvision.m3u8
+#EXTINF:-1 tvg-id="NukaTV.ec",Ñuka TV (720p)
+https://cloudvideo.servers10.com:8081/8118/index.m3u8
+#EXTINF:-1 tvg-id="OromarTV.ec",Oromar TV (720p)
+https://stream.oromartv.com:8082/hls/oromartv_hi/index.m3u8
```

Comment:
https://stream.oromartv.com:8082/hls/oromartv.m3u8 ?

the "low" quality doesn't load, the rest does.

Reply:
I know, that's kinda weird lol